unlock-app/src/hooks/useSocials.ts

```diff
@@ -0,0 +1,133 @@
+import { useQuery } from '@airstack/airstack-react'
+
+interface Social {
+  id: string
+  profileName: string
+  profileImage: string
+  profileDisplayName: string
+  profileUrl: string
+}
+interface Socials {
+  [address: string]: Social
+}
+
+const query = `query DomainsAndSocials($wallets: [Address!]) {
+  Domains(
+    input: {
+      filter: {
+        isPrimary: {
+          _eq: true
+        }, 
+        resolvedAddress: {
+          _in: $wallets
+        }
+      }, 
+      blockchain: ethereum
+    }) {
+    Domain {
+      resolvedAddress
+      name
+    }
+  }
+  Socials(
+    input: {
+      filter: {
+        userAssociatedAddresses: {
+          _in: $wallets
+        }
+      }, 
+      blockchain: ethereum
+    }) {
+    Social {
+      dappName
+      profileName
+      profileImageContentValue {
+        image {
+          original
+        }
+      }
+      profileUrl
+      userAssociatedAddresses
+      blockchain
+      chainId
+      dappName
+      coverImageURI
+      dappSlug
+      dappVersion
+      fnames
+      followerCount
+      followerTokenAddress
+      followingCount
+      handleTokenAddress
+      handleTokenId
+      id
+      identity
+      isDefault
+      isFarcasterPowerUser
+      location
+      metadataURI
+      profileBio
+      profileCreatedAtBlockNumber
+      profileCreatedAtBlockTimestamp
+      profileDisplayName
+      profileHandle
+      profileImage
+      profileLastUpdatedAtBlockNumber
+      profileLastUpdatedAtBlockTimestamp
+      profileMetadata
+      profileName
+      profileTokenAddress
+      profileTokenId
+      profileTokenIdHex
+      profileTokenUri
+      profileUrl
+      twitterUserName
+      updatedAt
+      userAddress
```

Comment:
are all these fields needed ?

Reply:
ha good point. not really. I will clean them up!